From: Henry Su <henry.su@amd.com>
Date: Fri, 11 May 2007 05:48:51 +0000 (-0700)
Subject: add the ATI SB700 SATA controller device id to AHCI pci table
X-Git-Tag: v2.6.22~696^2~2
X-Git-Url: http://git.kernel.org/?p=linux%2Fkernel%2Fgit%2Ftorvalds%2Flinux-2.6.git;a=commitdiff_plain;h=2bcfdde6767f2f07891d2753c25220012fe5e6d2

add the ATI SB700 SATA controller device id to AHCI pci table

Add the device ID to AHCI pci table for ATI SB700 SATA controller, the
subsequent chipset of SB600.

Signed-off-by: henry su<henry.su@amd.com>
Cc: Jeff Garzik <jeff@garzik.org>
Cc: Tejun Heo <htejun@gmail.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Jeff Garzik <jeff@garzik.org>
---

Backported to Debian's 2.6.18 by dann frazier <dannf@debian.org>

diff -urpN linux-source-2.6.18.orig/drivers/scsi/ahci.c linux-source-2.6.18/drivers/scsi/ahci.c
--- linux-source-2.6.18.orig/drivers/scsi/ahci.c	2007-05-23 08:49:57.000000000 -0600
+++ linux-source-2.6.18/drivers/scsi/ahci.c	2007-08-03 01:41:46.646787197 -0600
@@ -347,6 +347,8 @@ static const struct pci_device_id ahci_p
 	  board_ahci }, /* ATI SB600 non-raid */
 	{ PCI_VENDOR_ID_ATI, 0x4381, PCI_ANY_ID, PCI_ANY_ID, 0, 0,
 	  board_ahci }, /* ATI SB600 raid */
+	{ PCI_VENDOR_ID_ATI, 0x4390, PCI_ANY_ID, PCI_ANY_ID, 0, 0,
+	  board_ahci }, /* ATI SB700 */
 
 	/* VIA */
 	{ PCI_VENDOR_ID_VIA, 0x3349, PCI_ANY_ID, PCI_ANY_ID, 0, 0,
